src/modules/.internal/polyfills/index.ts

```diff
@@ -0,0 +1,19 @@
+const loadPolyfills = async () => {
+  // Check if the Intl object and Intl.NumberFormat function are available
```

Comment:
What is the criteria on injecting polyfill ? 
By below logic, polyfill gets loaded even if Intl and Intl.NumberFormat are absent

Reply:
the condition needs to be updated here. 
Check if Intl is present and NumberFormat is a function and formatToParts is absent.